lib/primitives/base-element.js

```diff
@@ -132,6 +135,19 @@ module.exports = function(registry) {
 
     setMetaProperty: function(name, value) {
       this.meta.set(name, value);
+    },
+
+    emitSwap: function() {
```

Comment:
I would handle passing the changed value in another event, like `update`. This is merely saying the branch of the tree has changed.